### PR TITLE
Update member organizations list

### PR DESCRIPTION
**Pull request by ddutta**
from Cisco

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/community/66)

### CODE REVIEW

File: members.yaml [hunk 1]
```diff
@@ -15,3 +15,10 @@
   lastName: Lewi
   affiliation: Google
   email: jlewi@google.com
+
+- github: ddutta
+  slack: debo
+  firstName: Debo
+  lastName: Dutta
+  affiliation: Cisco
+  email: dedutta@cisco.com
```
Comment (gaocegege) [inline members.yaml]: I think it has conflicts with your another PR #67 , maybe we could remove the changes here.

Comment (ddutta) [reply members.yaml]: thx ..

Comment (ddutta) [reply members.yaml]: I committed when the branch did have the conflict. Will resolve it ASAP

**Pull request closed without merging**